packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/tests/reducer.test.js

```diff
@@ -1390,6 +1473,7 @@ describe('CONTENT MANAGER | COMPONENTS | EditViewDataManagerProvider | reducer',
               "component": {
                 "__temp_key__": 1,
                 "field1": {
+                  "__temp_key__": 1,
```

Comment:
Should all these fields have the same `__temp_key__`? Could you run into e.g. keying problems because of that?